sportngin/aws_auditor
# First tests, Travis CI, MFA support, and fog file compatibility

Description and Impact
----------------------
Setup for rspec testing. 
Travis CI integration.
Update Ruby compatibility.
Add support for fog file credentials.
Add support for MFA.

QA Plan
-------
Verify you don't have a `.aws.yml` file in the project directory or above.
Run `bin/aws-auditor inspect [profile]` on a non-mfa and mfa profile.
